### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by adamchainz**
* Oxford comma applied - makes some clauses clearer
* Headline capitalisation for more "umph"
* Clarifying potential complaint reactions

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -33,42 +33,43 @@ contain:
...
-### Be aware
+### Be Aware
 
 We ask everyone to be aware that we will not tolerate intimidation, harassment,
-or any abusive, discriminatory or derogatory behaviour by anyone at any event
+or any abusive, discriminatory, or derogatory behaviour by anyone at any event,
 or online. Complaints can be made to the [meetup
 organisers](http://www.meetup.com/The-London-Django-Meetup-Group/members/?op=leaders)
-either face to face at a meetup, or via the Meetup website. All complaints
-made to event organisers will remain confidential and be taken seriously. The
```
Comment (adamchainz) [inline code_of_conduct.md]: This was a bit weird - mixing "all complaints" to "the complaint"

File: code_of_conduct.md [hunk 2]
```diff
@@ -33,42 +33,43 @@ contain:
...
-- be prohibited from continuing to attend the event
-- have the incident reported to the appropriate authorities
+- told to apologise
+- told to stop/modify their behaviour appropriately
+- warned that enforcement action may be taken if the behaviour continues
+- asked to leave the venue immediately
+- prohibited from continuing to attend the event
 
-### What does that mean for me?
+When necessary, an incident will be reported to the appropriate authorities.
```
Comment (adamchainz) [inline code_of_conduct.md]: I separated this as it made the bullet point list simply a list of actions against complainees, and indicated that is a "when necessary"/last resort kind of action.

**Comment by jamescooke** [comment]
On it.

File: code_of_conduct.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # London Django Meetup Code of Conduct
 
-## Be awesome to each other
+## Be Awesome to Each Other
 
 The mission of [The London Django Meetup
 Group](http://www.meetup.com/The-London-Django-Meetup-Group/) is to provide
-fun, welcoming and professional environments so that diverse groups of people­-
+fun, welcoming and professional environments so that diverse groups of people -
```
Comment (jamescooke) [inline code_of_conduct.md]: I'd like to clean the whitespace at the end of this line.

**Comment by jamescooke** [comment]
Thanks @adamchainz :+1:

**Pull request merged**